platform/api/aws/network.go

```diff
@@ -315,14 +315,21 @@ func (a *API) getSubnetID(vpc string) (string, error) {
 		},
 	})
 	if err != nil {
-		return "", fmt.Errorf("unable to get subnets for vpc %v: %v", vpc, err)
+		return nil, fmt.Errorf("unable to get subnets for vpc %v: %v", vpc, err)
 	}
+
+	var ids []string
+
 	for _, id := range subIds.Subnets {
 		if id.SubnetId != nil {
-			return *id.SubnetId, nil
+			 ids = append(ids, *id.SubnetId)
```

Comment:
NIT: an unnecessary blank before `ids`.
`gofmt` is your friend. ;-)

Reply:
Fixed.